### ISSUE TITLE
[BUG] Version increment workflow was not triggered 

### ISSUE THREAD

**Issue by dblock** [opensearch-project/OpenSearch#2013]
**Describe the bug**

Version increment workflow introduced in #1816 was not triggered. An identical workflow was triggered in common-utils (https://github.com/opensearch-project/common-utils/actions/workflows/version.yml, https://github.com/opensearch-project/common-utils/actions/runs/1741497956), but wasn't in OpenSearch and job-scheduler either.

**Expected behavior**

Auto-increment workflows should have run when we tagged 1.2.4.

**Comment by dblock** [comment]
@downsrob @saratvemulapalli @peterzhuamazon How did you create https://github.com/opensearch-project/job-scheduler/releases/tag/1.2.4.0 vs. https://github.com/opensearch-project/OpenSearch/releases/tag/1.2.4 vs. https://github.com/opensearch-project/common-utils/releases/tag/1.2.4.0?

**Comment by saratvemulapalli** [comment]
@dblock I manually created the release, the tag did already exist. Just used the github release drafter to release it.

**Comment by dblock** [comment]
@abhinavGupta16 Did you create tags manually for 1.2.4?

**Comment by abhinavGupta16** [comment]
No, we used https://github.com/opensearch-project/opensearch-build/issues/378#issuecomment-999700848 script to create tags for 1.2.4

@peterzhuamazon

**Comment by abhinavGupta16** [comment]
> @abhinavGupta16 Did you create tags manually for 1.2.4?

No, we used https://github.com/opensearch-project/opensearch-build/issues/378#issuecomment-999700848 script to create tags for 1.2.4

cc - @peterzhuamazon
